OpenRA.Renderer.Sdl2/Sdl2GraphicsDevice.cs

```diff
@@ -330,6 +330,43 @@ public void SetLineWidth(float width)
 			ErrorHandler.CheckGlError();
 		}
 
+		public Bitmap TakeScreenshot()
+		{
+			var rect = new Rectangle(Point.Empty, size);
+			var bitmap = new Bitmap(rect.Width, rect.Height, System.Drawing.Imaging.PixelFormat.Format32bppArgb);
+			var data = bitmap.LockBits(rect,
+				System.Drawing.Imaging.ImageLockMode.WriteOnly, System.Drawing.Imaging.PixelFormat.Format32bppArgb);
+
+			try
```

Comment:
I think I confused this with being the equivalent of `using` or copy pasted this from example bitmap code. Removed all this try-finally and unsafe stuff now.